### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by lucas-crodrigues**
This is the fifth part of the Blog app initiated in the first week of the Rails module in the Microverse program.

The requirements are: 

> ### Project requirements
> - Add RSpec as a gem to your project.
> - Create a Request spec file for all your controllers.
> - Make sure that for each action you check:
>     - If response status was correct.
>     - If a correct template was rendered.
>     - If the response body includes correct placeholder text.

- [x] [Add RSpec as a gem and update bundle](https://github.com/lucas-crodrigues/rails-blog/commit/eb7caaab8bd0585a763cf34fabc4ecc31a00414b).
- [x] [Write tests for Users](https://github.com/lucas-crodrigues/rails-blog/commit/ee29c14c41b12b2cc95c44176e894c02986ea66b).
- [x] [Write tests for Posts](https://github.com/lucas-crodrigues/rails-blog/commit/47701d6e36ba84c80220c3ce7f1753da180925c7).

Thank you.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -77,5 +77,6 @@ end
 gem 'ffi'
 
 group :test, :development do
+  gem 'rails-controller-testing'
   gem 'rspec-rails', '~> 6.0.0'
```
Comment (mateo951) [inline Gemfile:81]: Has and uses RSpec gem for testing ✔

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -2,6 +2,33 @@
```
Comment (mateo951) [inline spec/requests/posts_spec.rb:2]: Has `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory 🔥

**Review by mateo951 — APPROVED** [review]
### Hi @lucas-crodrigues, 

**🔥 Highlights 🔥**
- `index` and `show` actions for `User` and `Post` controllers are tested 💣

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Gemfile (hunk 1 above)
Comment (lucas-crodrigues) [reply Gemfile:81]: Thanks for the feedback mate 🤙

File: spec/requests/posts_spec.rb (hunk 2 above)
Comment (lucas-crodrigues) [reply spec/requests/posts_spec.rb:2]: (☞ﾟヮﾟ)☞

**Pull request merged**